server/src/service/card/card.service.ts

```diff
@@ -0,0 +1,48 @@
+import { inject, injectable } from 'inversify';
+
+import { Card } from '../../models/new-card';
+import { CardRepository } from './card.repository';
+import { LoggerService } from '../logger';
+
+@injectable()
+export class CardService {
```

Comment:
For what purpose you created this service, it is just unnecessary layer with try/catch and logging.
You already have logging on repository level, and try/catch in controller
Please remove this service